### PR TITLE
Security level sounds [MODULAR]

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## About The Pull Request

 adds several security level sounds. Like before. And also in a modular way...

## Why It's Good For The Game

Beep beep beep beep, station's gonna blow.

## Changelog
:cl:
add: Security level sounds.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/alerts/code/set_security_level.dm [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+			level = SEC_LEVEL_BLUE
+		if("red")
+			level = SEC_LEVEL_RED
+		if("delta")
+			level = SEC_LEVEL_DELTA
+
+	if(level >= SEC_LEVEL_GREEN && level <= SEC_LEVEL_DELTA && level != GLOB.security_level)
+		switch(level)
+			if(SEC_LEVEL_BLUE)
+				sound_to_playing_players('modular_skyrat/modules/alerts/sound/misc/notice1.ogg', volume = 50)
```
Comment (Azarak) [inline modular_skyrat/modules/alerts/code/set_security_level.dm]: Shouldn't this play the voyblue sound? And amber alert should play the notice1? You'll need to include the voyblue sound too

Comment (Gandalf2k15) [reply modular_skyrat/modules/alerts/code/set_security_level.dm]: Ah yeah lemme do that

File: code/modules/security_levels/security_levels.dm [hunk 2]
```diff
@@ -17,6 +17,8 @@ GLOBAL_VAR_INIT(security_level, SEC_LEVEL_GREEN)
 		if("delta")
 			level = SEC_LEVEL_DELTA
 
+	set_security_level_skyrat(level) //SKYRAT EDIT ADDITION - ALERTS
+
 	//Will not be announced if you try to set to the same level as it already is
 	if(level >= SEC_LEVEL_GREEN && level <= SEC_LEVEL_DELTA && level != GLOB.security_level)
 		switch(level)
```
Comment (Azarak) [inline code/modules/security_levels/security_levels.dm]:
```suggestion
	//Will not be announced if you try to set to the same level as it already is
	if(level >= SEC_LEVEL_GREEN && level <= SEC_LEVEL_DELTA && level != GLOB.security_level)
		set_security_level_skyrat(level) //SKYRAT EDIT ADDITION - ALERTS
		switch(level)
```

File: modular_skyrat/modules/alerts/code/set_security_level.dm [hunk 3]
```diff
@@ -0,0 +1,10 @@
+/proc/set_security_level_skyrat(level)
+	switch(level)
+		if(SEC_LEVEL_BLUE)
+			sound_to_playing_players('modular_skyrat/modules/alerts/sound/misc/voyalert.ogg', volume = 50)
+		if(SEC_LEVEL_RED)
+			sound_to_playing_players('modular_skyrat/modules/alerts/sound/misc/tas_red_alert.ogg', volume = 50)
+		if(SEC_LEVEL_DELTA)
+			sound_to_playing_players('modular_skyrat/modules/alerts/sound/misc/deltaklaxon.ogg')
+		else
+			return
```
Comment (Azarak) [inline modular_skyrat/modules/alerts/code/set_security_level.dm]:
```suggestion
```

File: modular_skyrat/modules/alerts/code/set_security_level.dm [hunk 4]
```diff
@@ -0,0 +1,10 @@
+/proc/set_security_level_skyrat(level)
+	switch(level)
+		if(SEC_LEVEL_BLUE)
+			sound_to_playing_players('modular_skyrat/modules/alerts/sound/misc/voyalert.ogg', volume = 50)
+		if(SEC_LEVEL_RED)
+			sound_to_playing_players('modular_skyrat/modules/alerts/sound/misc/tas_red_alert.ogg', volume = 50)
```
Comment (Azarak) [inline modular_skyrat/modules/alerts/code/set_security_level.dm]: Personally I like the old one more

**Pull request merged**